Dockerfile

```diff
@@ -5,13 +5,32 @@ RUN sed -i 's/ universe/ universe multiverse/' /etc/apt/sources.list
 
 RUN apt-get update && \
   apt-get dist-upgrade --yes && \
-  apt-get install --yes \
-  git \
-  snapcraft \
-  && \
   apt-get autoclean --yes && \
   apt-get clean --yes
 
+# Install snapcraft dependencies
+RUN apt-get install git sudo libslang2-dev python-pip python3-pip gcc g++ make python3-dev python3-venv libffi-dev \
+  libsodium-dev libapt-pkg-dev libarchive13 squashfs-tools patchelf --yes && \
+  apt-get autoclean --yes && \
+  apt-get clean --yes
```

Comment:
You might get a smaller image if you combine this RUN with the one above and add `rm -rf /var/lib/apt/lists/*` at the end.

Reply:
Removing those files seems to break things since snapcraft makes use of apt when it runs so I'm going to skip it.